pyproject.toml

```diff
@@ -32,6 +32,7 @@ classifiers = [
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
Should we keep using 3.10 separate and for the future? Once Airflow supports 3.10 we can add this, and start running our unit tests for a matrix for 3.7 to 3.10

Reply:
Oh, I didn't realise Airflow didn't support 3.10 - I'll remove this.